README.md

```diff
@@ -126,6 +126,25 @@ however, you can utilize any of the options above _in tandem_ with
 can be useful for setups with multiple configs that share common options for
 `webpack-serve`, but require subtle differences.
 
+### `webpack.config.js` example
+```js
+  const path = require('path');
+
+  const { NamedModulesPlugin } = require('webpack');
```

Comment:
Please remove this as this plugin isn't needed anymore since `webpack >= v4.0.0`, which will cause confusion for new users :)

Reply:
Fixed, and good tip, I was unaware of that.